tests/test_utils.py

```diff
@@ -1,13 +1,19 @@
+import aiofiles
```

Comment:
If you are going to use `aiofiles` for just test cases, I don't think it's necesary.
Since each test case anyway runs serially, we can just safely use the vanilla synchronous file I/O APIs from Python to validate the results.